controllers/release/adapter_test.go

```diff
@@ -1275,27 +1283,73 @@ var _ = Describe("Release adapter", Ordered, func() {
 				}
 			})
 
-			It("returns an error if the ReleasePlan has no author", func() {
+			It("returns invalid and an error if automated label is present but is not set in release status", func() {
+				valid, err := adapter.validateAuthor()
+				Expect(!valid).To(BeTrue())
+				Expect(err).To(HaveOccurred())
+				for i := range adapter.release.Status.Conditions {
+					if adapter.release.Status.Conditions[i].Type == "Validated" {
```

Comment:
Maybe we should expose the condition types so you can do `FindStatusCondition(adapter.release.Status.Conditions, v1alpha1.ReleaseValidatedConditionType)`. Probably we should create a ticket for the future to see if we want to tackle it.